JesusFilm/core
# feat: resource hub journeys

# Description

### Issue

We need a journeys page in the resources hub with components reused from journeys-admin.

[Link to Basecamp Todo](https://3.basecamp.com/3105655/projects)

### Solution

Add new journeys page in the watch project at the /journeys path
Add a sub page for viewing journey details at the /journeys/[journeysId] path

# External Changes

There should be no external changes to watch.

# Additional information

### How to Test

A user should be able to visit the journeys page at /journeys on the watch project
A user should be able to view and interact with the same journeys gallery as is in journeys-admin/templates
A user should be able to view the journeys template page at /journeys/[journeysId]
A user should be able to choose a template and be navigated to journeys-admin
